lms/static/js/fixtures/discovery.html

```diff
@@ -1,26 +1,28 @@
 <section class="courses-container">
-  <div id="discovery-form">
-    <form>
+  <div id="discovery-form" class="wrapper-search-context">
+    <div id="discovery-message" class="search-status-label"></div>
+    <form class="wrapper-search-input">
       <input class="discovery-input" placeholder="Search for a course" type="text"/><!-- removes spacing
       --><button type="submit" class="button postfix discovery-submit" aria-label="Search">
         <i class="icon fa fa-search" aria-hidden="true"></i>
+        <div aria-live="polite" aria-relevant="all">
+          <div id="loading-indicator" class="loading-spinner hidden">
+          <i class="icon fa fa-spinner fa-spin"></i>
+          <span class="sr">Loading</span>
+          </div>
+        </div>
       </button>
     </form>
-    <div id="discovery-message"></div>
-    <div aria-live="polite" aria-relevant="all">
-      <div id="loading-indicator" class="hidden">
-        <i class="icon fa fa-spinner fa-spin"></i> Loading
-      </div>
+    
+    <div id="filter-bar" class="filters hide-phone">
     </div>
-  </div>
 
-  <div id="filter-bar" class="filters hide-phone">
-  </div>
+    <section class="courses">
```

Comment:
I think this should just be a div - there are already too many nested sections and div matches the other siblings, .discovery-form and .filter-bar